swiper.el

```diff
@@ -514,6 +531,28 @@ such as `scroll-conservatively' are set to a high value.")
   "A predicate that decides whether `line-move' or `forward-line' is used.
 Note that `line-move' can be very slow.")
 
+(defun swiper--format-function (candidates)
+  "Format function for swiper CANDIDATES.
+
+Highlight current line number as in `display-line-numbers'."
+  (ivy--format-function-generic
+   (lambda (str)
+     (let ((ln (if swiper-include-line-number-in-search str
+                 (copy-sequence (get-text-property 0 'display str)))))
+       (when (and ln
+                  (eq (get-text-property 0 'face ln)
+                      'swiper-line-number-face))
+         (put-text-property
+          0 (next-single-property-change 0 'face ln (1- (length ln)))
```

Comment:
A comment explaining why 1 less than the `length` is needed would be nice.

Why is this `put-text-property` not predicated on `swiper-include-line-number-in-search` as well?

Reply:
The `put-text-property` here works to highlight the current line number in the overlay or in the string proper and doesn't interfere with the match faces in my testing if `swiper-include-line-number-in-search` is turned on. 

the `1-` was extraneous

Reply:
Right, I was confused about the purpose of `swiper-include-line-number-in-search`.

> the `1-` was extraneous

In that case you can remove the argument altogether and let `next-single-property-change` default to the full length.